docs/Makefile

```diff
@@ -8,6 +8,8 @@ SOURCEDIR     = source
 BUILDDIR      = build
 
 export PYTHONPATH := $(realpath ..):$(PYTHONPATH)
+# Add a sitecustomize.py to generate release notes
+export PYTHONPATH := $(realpath .)/prehook:$(PYTHONPATH)
```

Comment:
One downside is, though, it will always generates release notes even if we do, for example, `make clean`. But seems like it doesn't take a while (like a seconds).